build.ps1

```diff
@@ -1,7 +1,7 @@
 param(
     [string]$buildType="Debug",
     [string]$dotnetDir="c:\Program Files\dotnet",
-    [string]$msbuildDir="C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin",
+    [string]$msbuildDir="C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Current\Bin",
```

Comment:
Why 'Professional'?

Reply:
My mistake pushing the unwanted changes, fixed now.